From eb40f1a79042aa8a23fc86360adcaf9ee41a64d5 Mon Sep 17 00:00:00 2001
From: Ferdinand Xu <cheng.a.xu@intel.com>
Date: Thu, 16 Jul 2015 04:06:05 -0400
Subject: [PATCH 0652/1164] CDH-41244: HIVE-11243: Changing log level in
 Utilities.getBaseWork(Nemon Lou, reviewed by
 Ferdinand Xu)

(cherry picked from commit e2ee458d8ef417785f1e9ebfca303a9d15fee8a8)

Change-Id: I65999b51167d831d4e6e79b8681040d97ba505d8
---
 .../org/apache/hadoop/hive/ql/exec/Utilities.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java b/ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java
index af78672..d745b31 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java
@@ -449,7 +449,7 @@ private static BaseWork getBaseWork(Configuration conf, String name) {
       return gWork;
     } catch (FileNotFoundException fnf) {
       // happens. e.g.: no reduce work.
-      LOG.info("File not found: " + fnf.getMessage());
+      LOG.debug("File not found: " + fnf.getMessage());
       LOG.info("No plan file found: "+path);
       return null;
     } catch (Exception e) {
-- 
1.7.9.5

